### ISSUE TITLE
Implement HL7v2 Message Response in Submission API

### ISSUE THREAD

**Issue by arnejduranovic** [CDCgov/prime-reportstream#15731]
### User Story

As a sender of Lab Orders using the EPIC system,
I require ReportStream's submission API to return an HL7v2 formatted response instead of the normal JSON,
so that the EPIC system can properly acknowledge the submission was successful. 

### Description/Use Case

Certain senders may require RS response objects to be in a certain format, which may potentially limit the information RS can provide. This ticket is specifically asking for a  HL7v2 formatted message to support epic systems sending lab orders. It is likely this HL7v2 message will be the following: https://hl7-definition.caristix.com/v2/HL7v2.5.1/Segments/MSA

### Dev Notes

- The submission API in RS returns a JSON object with a lot of properties, and we may not need to port all of them to the new message type. This is pending requirements.

- Epic has indicated they are willing to populate “x-application” with a value of “hl7-v2+er7” to indicate to RS it should return an HL7v2 message in the body of the response. Another option is we can create a new sender setting that indicates what format senders want their responses it.

### Acceptance Criteria

- [ ] Submission endpoint supports returning HL7v2 formatted body in HTTP response

**Comment by arnejduranovic** [comment]
Waiting on ETOR-TI team to provide exact requirements for the HL7v2 message for this use case.

**Comment by JohnNKing** [comment]
Draft ETOR ACK requirements: https://cdc.sharepoint.com/:w:/r/teams/OLSS_ETOR_ReportStream/Shared%20Documents/ReportStream%20ETOR%20ACK%20Requirements.docx?d=w93bd416399924de6a301626414d6cea2&csf=1&web=1&e=OEaZKR

Feedback during the LA SME call today indicated we were on the right track. However, we may receive more partner feedback in the coming week.

**Comment by JohnNKing** [comment]
New version of the requirements doc: https://cdc.sharepoint.com/:w:/r/teams/OLSS_ETOR_ReportStream/_layouts/15/Doc.aspx?sourcedoc=%7BE5A9FAE6-8CD2-4C99-A757-F027562C1737%7D&file=ReportStream%20ETOR%20ACK%20Requirements.docx&nav=eyJjIjoxNjg2ODc2OTM0fQ&action=default&mobileredirect=true

CDC is reaching out to the hospital to confirm they have no additional feedback.

**Comment by JohnNKing** [comment]
Feedback from one of our partners (Ochsner): ACKs in staging are working as expected, for orders sent from Epic! 🎉

**Comment by arnejduranovic** [comment]
Duplicate of: https://github.com/CDCgov/prime-reportstream/issues/16394
